react-components/src/analysis/view/AnalysesView.js

```diff
@@ -947,9 +934,14 @@ class AnalysesView extends Component {
                 <Prompt analysis={selectedAnalysis}
                         intl={intl}
                         renameDialogOpen={this.state.renameDialogOpen}
-                        commentsDialogOpen={this.state.commentsDialogOpen}
                         onRenameOkClicked={this.doRename}
                         onRenameCancelClicked={() => this.setState({renameDialogOpen: false})}
+                />
+                }
+                {selectedAnalysis &&
+                <Prompt analysis={selectedAnalysis}
```

Comment:
I still think you can get rid of the `Prompt` function and replace it with `DEPromptDialog` components here.